packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -46,49 +49,74 @@ export class AmplifyProjectCreator {
 
-    await printer.indicateProgress(
-      `Installing required dependencies`,
-      async () => {
-        await this.packageManagerController.installDependencies(
-          this.defaultProdPackages,
-          'prod'
-        );
+    printer.printNewLine();
+    printer.log(format.sectionHeader(`Installing devDependencies:`));
+    printer.log(format.list(this.defaultDevPackages));
+
+    printer.printNewLine();
+    printer.log(format.sectionHeader(`Installing dependencies:`));
+    printer.log(format.list(this.defaultProdPackages));
+    printer.printNewLine();
 
-        await this.packageManagerController.installDependencies(
+    await printer.indicateProgress(
+      () =>
+        this.packageManagerController.installDependencies(
           this.defaultDevPackages,
           'dev'
-        );
-      }
+        ),
+      'Installing devDependencies...',
+      'DevDependencies installed'
     );
 
-    await printer.indicateProgress(`Creating template files`, async () => {
-      await this.gitIgnoreInitializer.ensureInitialized();
+    await printer.indicateProgress(
+      () =>
+        this.packageManagerController.installDependencies(
+          this.defaultProdPackages,
+          'prod'
+        ),
+      'Installing dependencies...',
+      'Dependencies installed'
+    );
 
-      await this.initialProjectFileGenerator.generateInitialProjectFiles();
-    });
+    await printer.indicateProgress(
+      async () => {
+        await this.gitIgnoreInitializer.ensureInitialized();
+        await this.initialProjectFileGenerator.generateInitialProjectFiles();
+      },
+      'Creating template files...',
+      'Template files created'
+    );
 
-    printer.log('Successfully created a new project!');
+    printer.log(green('Successfully created a new project!'));
+    printer.printNewLine();
 
     const cdPreamble =
       process.cwd() === this.projectRoot
-        ? ''
-        : `Navigate to your project directory using
-'cd .${this.projectRoot.replace(process.cwd(), '')}'.
-Then get started with the following commands:
-`;
-
+        ? null
+        : `Navigate to your project directory using ${cyan(
+            bold('cd .' + this.projectRoot.replace(process.cwd(), '') + '')
+          )} and then:`;
+
+    printer.log(blue(bold(`Welcome to AWS Amplify!`)));
+
+    const instructionSteps = [
+      cdPreamble,
+      this.packageManagerController.getWelcomeMessage(),
+    ]
+      .filter(Boolean)
+      .join(EOL);
+
+    printer.log(instructionSteps);
+    printer.printNewLine();
     printer.log(
-      `Welcome to AWS Amplify! 
-${cdPreamble}
-${this.packageManagerController.getWelcomeMessage()}`
-    );
-
-    printer.log(
-      `Amplify (Gen 2) collects anonymous telemetry data about general usage of the CLI.
-
-Participation is optional, and you may opt-out by using \`amplify configure telemetry disable\`.
-
-To learn more about telemetry, visit ${LEARN_MORE_USAGE_DATA_TRACKING_LINK}`
+      grey(
```

Comment:
This should be another format utility like `format.note()` or something like that